storage/innobase/xtrabackup/test/t/check_for_passwords.sh

```diff
@@ -0,0 +1,93 @@
+###############################################################################
+# check if for any password store in data dir thread info or in history table
+###############################################################################
+
+###############################################################################
+
+. inc/common.sh
+password_string=secretpwd
+
+MYSQLD_EXTRA_MY_CNF_OPTS="
+secure-file-priv=$TEST_VAR_ROOT
+"
+vlog "Preparing server"
+start_server
+mysql -e "CREATE USER pxb@localhost IDENTIFIED BY '$password_string'"
+mysql -e "GRANT ALL ON *.* TO pxb@localhost "
+###MyISAM table is used so PXB wait untill running update are complete
+mysql -e "CREATE TABLE t1(i int) engine=MYISAM" test
+mysql -e "INSERT INTO t1 values(1)" test
+mkdir -p $topdir/backup
+
+#pass option like password -p to be run with
+
+function grep_in_datadir_ps() {
+
+  additional_options=$1
+  ### This SLEEP with update on MyISAM will let backup wait untill this query is killed. PXB waits to acquire FLUSH TABLES on MyISAM tables
```

Comment:
The running update on myisam table will hold access to the table. PXB will do FLUSH TABLES on MyISAM tables which requires access to the table and waits for the running update on MyISAM to finish.

Reply:
fixed